iCubGenova09/hardware/mechanicals/left_arm-eb1-j0_1-mec.xml

```diff
@@ -12,16 +12,16 @@
         <param name="fullscalePWM">             32000               32000               </param>
         <param name="ampsToSensor">             1000.0              1000.0              </param>
         <param name="Gearbox_M2J">              100.00              100.00              </param>
-        <param name="Gearbox_E2J">              1                   1                   </param>
+        <param name="Gearbox_E2J">              64                  64                   </param>
         <param name="useMotorSpeedFbk">         1                   1                   </param>
         <param name="MotorType">                "MOOG-BL-C2900576"  "MOOG-BL-C2900576"  </param>
         <param name="Verbose"> 0 </param>
     </group>
     
     <group name="LIMITS">
         <!--                                    0                   1              -->
-        <param name="hardwareJntPosMax">        25                165                 </param>
-        <param name="hardwareJntPosMin">       -180                 0                 </param>
+        <param name="hardwareJntPosMax">        36.2                165                 </param>
```

Comment:
Is it this wide? From https://github.com/icub-tech-iit/tickets/issues/1410#issuecomment-953972983 it should be +25 - 180

Reply:
36.2 is the measured angle of the hard stop; this is necessary to let the incremental procedure work

cc @ale-git

Reply:
Measured how? Is it possible that the encoder was miscalibrated? The mechanical range of motion should be 205deg

Reply:
moving by hand to the hard stop and reading the motorgui.
@ale-git I'm right?